app/tests/spec/lib/fxa-client.js

```diff
@@ -743,25 +765,45 @@ define(function (require, exports, module) {
                 ));
           });
       });
+
+      it('passes along an optional `metricsContext`', function () {
+        return client.passwordReset(email, relier, { metricsContext: {}, resume: resumeToken })
+          .then(function () {
+            return client.passwordResetResend(email, passwordForgotToken, relier, { metricsContext: {}, resume: resumeToken });
```

Comment:
Can you break up this line at ~ 100-120 chars? It scrolls off the right hand side of the GitHub review screen.